Homework10/Homework10/Entities/LargeHomeApplianceEntity.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Homework10.Enums;
+
+namespace Homework10.Entities
+{
+	public class LargeHomeApplianceEntity : ElectricalApplianceEntity
+	{
+        public SizeType Size { get; set; }
```

Comment:
I think this Property should be added to ElectricalApplianceEntity/ElectricalAppliance as well and in the child class we will just hardcode this type